README.md

```diff
@@ -61,3 +61,27 @@ groovy order.groovy bench
 cd typescript
 yarn run bench
 ```
+
+## Ethereum and Starknet Keys
+
+Ethereum (L1) private key is used to sign a signature. This signature is deterministic
+and can only be generated by the owner of the private key.
+
+This signature is then used to generate Starknet (L2) private key
+(also referred to as Paradex private key).
+
+Starknet (L2) private key can used to deduce Starknet public key.
+This private key is also used to sign auth, onboarding and order signatures.
+
+Auth endpoint also requires providing the Ethereum address which is derived
+from the Ethereum private key in examples of some languages.
+
+During the onboarding process, an Account Contract is deployed on Starknet
+with the Starknet public key. This contract is used to store the user's balances
+and this is what you see in UI as your wallet.
```

Comment:
> and this is what you see in UI as your wallet

What is `this` referring to? It seems that it's referring to the public key. The public key however is not available on the UI. What is available is the contract address. If not referring to the public key, could you clarify?